src/plugins/visualizations/public/plugin.ts

```diff
@@ -179,7 +180,7 @@ export class VisualizationsPlugin
 
   public start(
     core: CoreStart,
-    { data, expressions, uiActions, embeddable, dashboard }: VisualizationsStartDeps
+    { data, expressions, uiActions, embeddable, dashboard, visAugmenter }: VisualizationsStartDeps
```

Comment:
We cannot take a hard dependency on `visAugmenter` this way or it causes a circular dependency. This is why we need to manually re-create the loader ourselves in `start()`

Reply:
Fixed